### ISSUE TITLE
Modify usage examples to be less NETCONF focused

### ISSUE THREAD

**Issue by kwatsen** [netmod-wg/yang-next#13]
The Usage Example sections throughout the spec illustrate NETCONF usage.   Either these examples should be updated to be equal parts NETCONF and RESTCONF, or they should be removed entirely.

**Comment by kwatsen** [comment]
I'm beginning to become a fan of the idea of defining a generic
"Requirements for Protocols Transporting YANG-modeled Data"
document - that would not only discuss security aspects, but
also generic protocol operations, that documents like NC, RC,
CoAP, etc. can point to...and even YANG (RFC 7950), rather than
pointing directly at NETCONF as it does today...

**Comment by schoenw** [comment]
I am OK with making the document less NETCONF specific but I am not OK with the idea to make the language overly generic (such as removing config true/false).

**Comment by mbj4668** [comment]
I don't think we should remove examples in our quest to keep the document protocol agnostic.  Examples are there to help the reader, for a reason.  And they are just examples, so I don't see the point in removing them.

**Comment by abierman** [comment]
do not support redoing the examples
